dev/src/codewind/cli/CLISetup.ts

```diff
@@ -313,9 +313,9 @@ namespace CLISetup {
 
             await Promise.all([
                 fs.rename(extractCwctlPath, getCwctlPath())
-                .then(() => {
-                    fs.chmod(getCwctlPath(), EXECUTABLES_MODE);
-                }),
+                    .then(() => {
```

Comment:
Can you undo all these whitespace changes?

Reply:
Done